### PR TITLE
Remove all deprecated methods

### PR DESCRIPTION
**Pull request by regiskuckaertz**
As 1.0 is nearing release, I think we can take a bold step and get rid of all deprecated methods. 

I had already written the ZIO tests using the table API so it was just a matter of copy-pasting and changing the interpreter 🎉 

What do you think @philwills ?

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
@TBonnin Great idea! See #276

**Comment by regiskuckaertz** [comment]
I wouldn't mind a review 🤗

File: alpakka/src/test/scala/com/gu/scanamo/ScanamoAlpakkaSpec.scala [hunk 1]
```diff
@@ -178,320 +184,300 @@ class ScanamoAlpakkaSpec extends FunSpecLike with BeforeAndAfterAll with Matcher
...
+  //         res3 <- bears.index(i).limit(1).scanFrom(res2._2)
+  //       } yield res2._1 ::: res3._1
+  //     } yield bs
+
+  //     ScanamoAlpakka.exec(alpakkaClient)(result).futureValue should equal(
+  //       List(Right(Bear("Yogi", "picnic baskets", Some("Kanga"))), Right(Bear("Pooh", "honey", Some("Winnie"))))
+  //     )
+  //   }
+  // }
+
```
Comment (TBonnin) [inline alpakka/src/test/scala/com/gu/scanamo/ScanamoAlpakkaSpec.scala]: why is this commenting out?

Comment (regiskuckaertz) [reply alpakka/src/test/scala/com/gu/scanamo/ScanamoAlpakkaSpec.scala]: good question. they were not working as expected with the `scanFrom` API; I will uncomment them once I've rebased [the PR](#274) that introduces `from` and `query0` 🤗

**Comment by regiskuckaertz** [comment]
Thank you!

**Pull request merged**